core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -387,7 +387,7 @@ func (ec *EthConfirmer) CheckForReceipts(ctx context.Context, blockNum int64) er
 		}
 	}
 
-	if err := ec.markAllConfirmedMissingReceipt(); err != nil {
```

Comment:
Is the `markAllConfirmedMissingReceipt` method removed?

Reply:
Its moved to the ORM: https://github.com/smartcontractkit/chainlink/pull/8467/files#diff-17acb6da51a93f6407f839210ff4ecd5809bc34eccc3252aa5e613429d4ca386R42